docker/control/Dockerfile

```diff
@@ -24,15 +23,21 @@ RUN wget https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein
     chmod +x /usr/bin/lein && \
     lein self-install
 
+ARG uid
+ARG gid
+
+RUN groupadd -g $gid kyle \
```

Comment:
Should other users be creating groups and users named "kyle"?

Reply:
You reckon "jepsen" is more appropriate? Or other preferences?

Reply:
I'd say "jepsen", but I... honestly don't understand exactly what context this code runs in. Are these users being created on the control node? DB node? The user's node, outside the docker env?

Reply:
This user will be added at the control node with same user and group id as the user on the host system. This means that when docker writes files it will write them with the same user and group id as your user rather than writing as root which is the default.

Does that make sense? (Yes, docker is crap.)

Reply:
Huh, okay, I think that makes sense! This *looks* okay to me, but Docker's really out of my depth, so I'm gonna ask a friend what she thinks. @mkcp, when you get a chance, could you give this patch a quick once-over and let me know what you think? I just wanna make sure we're being polite to other users. :)

Reply:
More precisely: when docker writes to the mounted volumes (which are shared with the host file system).